pkgs/stdenv/generic/check-meta.nix

```diff
@@ -117,6 +116,16 @@ let
   showLicense = showLicenseOrSourceType;
   showSourceType = showLicenseOrSourceType;
 
+  hasWarning = attrs: (attrs.meta.issues or []) != [];
+
+  ignoreWarningsDefaultPredicate = x: builtins.elem (getName x) (config.ignoreWarningsPackages or []);
```

Comment:
We are using `getName` here, which uses both the pname and the version. I wonder whether only using pname would be a more sensible default here.